.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+package-lock.json
```

Comment:
Any particular reason why you are ignoring the package-lock.json?
Was under the impression that this would give us a deterministic install of the project.

Reply:
You might be right. I am used to always ignore package-lock since it is auto-generated and I am thinking to have one source of the truth. But npm itself does recommend committing the file.